app/views/api/services/forms/_integration_settings.html.slim

```diff
@@ -2,14 +2,25 @@ javascript:
   $(document).ready(function(){
     // warning when changing auth mode
     var auth_mode_changed = false;
+    var current_auth_mode = $("input[name='service[backend_version]']:checked").val();
```

Comment:
Wouldn't it be better to check against the DB value than the HTML?
I think if you reload the page, some browser keeps the changed form values

Reply:
changed.